pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -312,6 +346,105 @@ func TestMemoryPressure_VerifyPodStatus(t *testing.T) {
 	}
 }
 
+func TestPIDPressure_VerifyPodStatus(t *testing.T) {
```

Comment:
This test case looks perfect!

Can you add another one similar to TestMemoryPressure?

It would be nice to mimic those tests for determining pressure, deleting pods, etc.

Take a look at those tests and see.

Reply:
@hrishikeshdkakkad any updates on this?

Reply:
Hello, since there hasn't been any progress on this for the past few months I cherry-picked @hrishikeshdkakkad commit and continued from there, please see: https://github.com/kubernetes/kubernetes/pull/123095